jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,533 @@
+---
+layout: classic-docs
+title: "CircleCI API Developer's Guide"
+short-title: "Developer's Guide"
+description: "API cookbook for internal and external CircleCI developers"
+categories: [getting-started]
+order: 1
+---
+
+This *API Developer's Guide* was written to assist developers in quickly and easily making API calls to CircleCI services to return detailed information about users, pipelines, projects and workflows.
+
+* TOC
+{:toc}
+
+# API Overview
+
+The CircleCI platform provides an API that enables users to interact with the CircleCI platform (e.g. triggering pipelines, updating settings etc, in addition to retrieving detailed information about users, jobs, workflows and pipelines.
+
+## API Classes
+
+The current categories of API v2 endpoints are:
+
+* Authentication
+* Pipeline
+* Workflows
```

Comment:
I think insights (preview) needs adding here?

Reply:
@rosieyohannan I intentionally left this out of the list because we are not exposing anything related to insights in this initial release.

Reply:
Ah ok, I only ask because I saw it's there in the public API docs